<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>606</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>131</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>321</width>
        <height>95</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="lineedit_username">
         <property name="placeholderText">
          <string>Please Enter Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineedit_password">
         <property name="inputMask">
          <string/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>Please Enter Password</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_login">
         <property name="text">
          <string>Log In</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLineEdit" name="lineedit_from">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>191</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="combobox_from">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="button_convert">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Convert</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineedit_to">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>10</y>
        <width>211</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="combobox_to">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>10</y>
        <width>61</width>
        <height>27</height>
       </rect>
      </property>
      <property name="maxVisibleItems">
       <number>10</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_from">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>221</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>GEL</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_to">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>40</y>
        <width>271</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>GEL</string>
      </property>
     </widget>
     <widget class="QPushButton" name="button_reset">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>87</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_status">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>75</y>
        <width>371</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="button_logout">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>100</y>
        <width>87</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Log Out</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>606</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
